### ISSUE TITLE
Charlie Discovery Day Project

### ISSUE THREAD

**Issue by Charlie-Henry** [cityofaustin/atd-data-tech#7779]
I'll be testing out implementing the dynamic legends + charts tutorial from Mapbox: https://www.mapbox.com/impact-tools/charts

If I have some extra time I'll try to bring this map into PowerBI: https://docs.mapbox.com/help/tutorials/power-bi/

**Comment by Charlie-Henry** [comment]
https://github.com/Charlie-Henry/chart-map-discovery

**Comment by Charlie-Henry** [comment]
I created a map that shows traffic volumes since COVID-19 using 5 [gridsmart camera's data.](https://data.austintexas.gov/Transportation-and-Mobility/Camera-Traffic-Counts/sh59-i6y9) The data is stored in a google sheets endpoint, but it seems to only work like 50% of the time. If I refresh a couple times it'll load the data.

https://charlie-henry.github.io/chart-map-discovery/

I also explored the Mapbox extension for Power BI. It was really easy to get set up with a custom basemap and add your own data on top.
